engine/src/cmd/energetic.h

```diff
@@ -66,33 +69,33 @@ class Energetic
     // TODO: move to StarFaring class when available
     struct UnitJump
     {
-        float warpDriveRating;
-        float energy;                            //short fix
-        float insysenergy;                       //short fix
-        signed char   drive;
-        unsigned char delay;
-        unsigned char damage;
+        float warpDriveRating = 0;
+        float energy = 100;                            //short fix
+        float insysenergy;  //short fix
+        signed char   drive = -2; // disabled
+        unsigned char delay = 5;
+        unsigned char damage = 0;
```

Comment:
this would be better in a constructor - and yes, `struct` can have constructors in C++.

Reply:
I am aware that structs and classes simply differ in the default visibility modifier, but in this case it's probably more relevant to do it in the outer class constructor anyhow. It's immediately used by the _jump_ field, so no other insntace can use it anyhow (not sure what this is called).

In any case, this is an excellent example of a struct that accumulated seemingly unrelated fields. It has ratings, energy and insysenergy (with a _short fix_ comment!) - all seem to be the same thing more or less. They had everything in unit_generic, including lots of variables, so their solution was to stick a couple in a struct.

Reply:
Also missed this.